### PR TITLE
Add github connection screenshot

### PR DESCRIPTION
**Pull request by unDemian**
#### Changes proposed in this Pull Request

Fixes 4601-gh-Automattic/p2 and 4434-gh-Automattic/p2 by adding GitHub connection screenshot
 
Markup 2022-02-10 at 3 34 59 PM

#### Testing instructions
- http://calypso.localhost:3000/marketing/connections/<hub-url> there should be a GitHub screenshot

*

Related to #

### CODE REVIEW

File: client/my-sites/marketing/connections/service-description.jsx [hunk 1]
```diff
@@ -148,13 +148,13 @@ class SharingServiceDescription extends Component {
 			},
 			p2_github: function () {
 				if ( this.props.numberOfConnections > 0 ) {
-					return this.props.translate( 'Workspace connected to Github.', {
-						comment: 'Embed Github Issues in P2 posts.',
+					return this.props.translate( 'Workspace connected to GitHub.', {
```
Comment (a8ci18n) [inline client/my-sites/marketing/connections/service-description.jsx:151]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/marketing/connections/service-description.jsx [hunk 2]
```diff
@@ -148,13 +148,13 @@ class SharingServiceDescription extends Component {
...
+						comment: 'Embed GitHub Issues in P2 posts.',
 					} );
 				}
 
-				return this.props.translate( 'Connect this workspace to your Github.', {
-					comment: 'Embed Github Issues in P2 posts.',
+				return this.props.translate( 'Connect this workspace to your GitHub.', {
```
Comment (a8ci18n) [inline client/my-sites/marketing/connections/service-description.jsx:156]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/marketing/connections/service-examples.jsx [hunk 3]
```diff
@@ -324,29 +324,13 @@ class SharingServiceExamples extends Component {
...
-					src: '/calypso/images/sharing/slack-screenshot-1.png',
-					alt: this.props.translate( 'Get Slack notifications with every new P2 post.', {
+					src: '/calypso/images/sharing/github-screenshot.png',
+					alt: this.props.translate( 'Get GitHub previews inside your P2 posts.', {
 						textOnly: true,
 					} ),
 				},
 				label: this.props.translate(
-					'Get {{strong}}Github previews{{/strong}} inside your P2 posts.',
+					'Get {{strong}}GitHub previews{{/strong}} inside your P2 posts.',
```
Comment (a8ci18n) [inline client/my-sites/marketing/connections/service-examples.jsx:333]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Review by nefeline — APPROVED** [review]
Works as described: confirmed the GH screenshot is now in place 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7140918

Thank you @unDemian for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7140918).

**Pull request merged**